### PR TITLE
installer/pkg/quota/gcp/limits.go: fix minor typo in comment

### PR DESCRIPTION
**Pull request by RiRa12621**

### CODE REVIEW

File: pkg/quota/gcp/limits.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 	serviceusage "google.golang.org/api/serviceusage/v1beta1"
 )
 
-// loadLimis loads the comumer quota metric for a given project and the list of services.
+// loadLimits loads the comumer quota metric for a given project and the list of services.
```
Comment (mtnbikenc) [inline pkg/quota/gcp/limits.go:11]: Maybe also fix another typo?
```suggestion
// loadLimits loads the consumer quota metric for a given project and the list of services.
```

**Comment by RiRa12621** [comment]
/retest
why is my change to the comments breaking all tests

**Review by jstuever — APPROVED** [review]

**Pull request merged**